src/sidebar/components/Annotation/AnnotationHeader.js

```diff
@@ -113,10 +105,21 @@ function AnnotationHeader({
     // an ID.
     store.setExpanded(/** @type {string} */ (annotation.id), true);
 
+  const group = store.getGroup(annotation.group);
```

Comment:
This is now passed as a prop to `AnnotationShareInfo` to allow `AnnotationShareInfo` to be dumber.